CHANGES.rst

```diff
@@ -1,6 +1,11 @@
 PyVISA-sim Changelog
 ====================
 
+Unreleased
+----------
```

Comment:
Changelog updated. Not sure how you like to run things. I typically build out an "Unreleased" section during WIP and then rename the section to the release version.

LMK if you want something else done.

Reply:
This will do